packages/mobile/src/features/videos/components/FiltersScreen/ShortcutToFilters.tsx

```diff
@@ -0,0 +1,212 @@
+const ImageEyeBeforeClick = styled.Image`
+    height: 14px;
+    width: 17px;
+    opacity: 0.2;
+`;
+
+const FilersButtonWrapper = styled.View`
+    display:flex;
+    flex-wrap: wrap;
+    margin-right: 16px;
+    flex-direction: row;
+`;
+
+
+interface ShortcutToFiltersProps {
+  navigation: any,
+  serachitem: string,
+  onSubmitSearch(): void;
+}
+
+export default class ShortcutToFilters extends React.Component <ShortcutToFiltersProps, any>{
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showEye : false,
+    };
+
+    this.callFunc = this.callFunc.bind(this);
+  }
+
+  callFunc() {
+    if (this.state.showEye) {
+      this.setState({showEye:false});
+    }else {
+      this.setState({showEye: true});      
+    }
+  }
+
+  static navigationOptions = {
+    headerShown: false,
+};
+
+  render() {
+    return (
+      <Wrapper>
+        <TitleEyeAndXWrapper>
+           <TitleAndEyeWrapper>
+              <SearchbarWrapper>
+                <InputText 
+                    autoCapitalize='none'
+                    autoCorrect={false} 
+                    placeholder="Docker od podstaw"
+                    value={this.props.serachitem}
+                    onChangeText={this.props.onSubmitSearch}                   
+                    onEndEditing={this.props.onSubmitSearch}                        
+                />
+                <SearchButton>
+                  <ImageSerach source={{uri:'magnifier'}}/>
+                </SearchButton>  
+              </SearchbarWrapper>   
+              <EyeButton onPress={this.callFunc}>
+                  {this.state.showEye === false && <ImageEyeBeforeClick source={{uri: 'eye'}} />}
+                  {this.state.showEye && <ImageEye source={{uri: 'eye'}} /> }
+              </EyeButton>
+            </TitleAndEyeWrapper>   
+            <XButton
+                navigation={this.props.navigation}
+                goto={NavigationName.HOME}/> 
+        </TitleEyeAndXWrapper>
+
+        <FilersButtonWrapper>
+            <BlackFilterButtonInShorcut 
+              text="Ux/UI Design"
+              navigation={this.props.navigation}
+              goto={NavigationName.HOME}
+              />
+              <BlackFilterButtonInShorcut 
+              text="Graphic Design"
+              navigation={this.props.navigation}
+              goto={NavigationName.HOME}
+              />
+              <BlackFilterButtonInShorcut 
+              text="Computer Science Engineer"
+              navigation={this.props.navigation}
+              goto={NavigationName.HOME}
+              />
+              <BlackFilterButtonInShorcut 
+              text="X"
+              navigation={this.props.navigation}
+              goto={NavigationName.HOME}
+              />
+              <BlackFilterButtonInShorcut 
+              text="ABCDEFGHIJK"
+              navigation={this.props.navigation}
+              goto={NavigationName.HOME}
+              />
+        </FilersButtonWrapper>
```

Comment:
wydaje mi się że ten komponent jest zbyt duży, a jeżeli chodzi o przyciski to dane kategori już są zwrócone.